integr8ly/integreatly-operator
# Intly 2342 codeready backups

Verification:
- Create a test namespace
- Create 3scale secrets:
  - `oc process -f https://gist.githubusercontent.com/philbrookes/b35a9571f326fcdeb8e3270772a35839/raw/b7ac6da42c7f576cebe82fbf3a2dfa1ef5974e5d/gistfile1.txt -p INSTALLATION_NAMESPACE=<test-namespace> -p AWS_ACCESS_KEY_ID=<access key> -p AWS_SECRET_ACCESS_KEY=<access secret> -p AWS_BUCKET=<test bucket> -p AWS_REGION=eu-central-1  | oc apply -f -`
- Delete any existing operator sources for integreatly
- Create operator source pointing at philbrookes
- Subscribe to integreatly 1.3.10 from operatorhub into the created namespace.
- Create installation CR
- Wait for it to complete
- See cronjobs are created
### Bonus verification
- Copy cronjob to a job and see it run successfully
- See the postgres data backed up in the defined s3 bucket.
